docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Remote WebDriver client"
+weight: 2
+---
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Diese Seite wird von Englisch 
+auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
+zu übersetzen indem Du uns einen Pull Reqeust schickst!
+ {{% /notice %}}
+
+To run a remote WebDriver client, we first need to connect to the RemoteWebDriver.
+We do this by pointing the URL to the address of the server running our tests.
+In order to customize our configuration, we set desired capabilities.
+Below is an example of instantiating a remote WebDriver object
+pointing to our remote web server, _www.example.com_,
+running our tests on Firefox.
+
+{{< code-tab >}}
+  {{< code-panel language="java" >}}
+FirefoxOptions firefoxOptions = new FirefoxOptions();
+WebDriver driver = new RemoteWebDriver(new URL("http://www.example.com"), firefoxOptions);
+driver.get("http://www.google.com");
+driver.quit();
+  {{< / code-panel >}}
+  {{< code-panel language="python" >}}
+# We don't have a Python code sample yet -  Help us out and raise a PR  
+  {{< / code-panel >}}
+  {{< code-panel language="csharp" >}}
+ FirefoxOptions firefoxOptions = new FirefoxOptions();
+ IWebDriver driver = new RemoteWebDriver(new Uri("http://www.example.com"), firefoxOptions);
+ driver.Navigate().GoToUrl("http://www.google.com");
+ driver.Quit();
+  {{< / code-panel >}}
+  {{< code-panel language="ruby" >}}
+require 'selenium-webdriver'
+
+driver = Selenium::WebDriver.for :remote, url: "http://www.example.com", desired_capabilities: :firefox
+driver.get "http://www.google.com"
+driver.close
+  {{< / code-panel >}}
+  {{< code-panel language="javascript" >}}
+// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+  {{< / code-panel >}}
+  {{< code-panel language="kotlin" >}}
+firefoxOptions = FirefoxOptions()
+driver: WebDriver = new RemoteWebDriver(new URL("http://www.example.com"), firefoxOptions)
+driver.get("http://www.google.com")
+driver.quit()
+  {{< / code-panel >}}
+{{< / code-tab >}}
+
+
+To further customize our test configuration, we can add other desired capabilities.
+
+
+## Desired capabilities
```

Comment:
This is also out of date. I believe the fork was not updated with the recent changes.

Reply:
I updated my PR, see comment below.